OneSchedule.Domain/StateMachine/AskState/AskBeginDateState.cs

```diff
@@ -0,0 +1,27 @@
+﻿using OneSchedule.Attributes;
+using OneSchedule.Domain.Abstractions.StateMachine;
+using OneSchedule.Domain.Models;
+using System.Threading.Tasks;
+using Telegram.Bot;
+
+namespace OneSchedule.Domain.StateMachine.AskState
```

Comment:
Move all the logic of Ask... states to the base class. All these classes just duplicate the same logic. Remove those duplications of code